docs/release-notes/8.16.asciidoc

```diff
@@ -56,6 +56,28 @@ On November 12, 2024, it was discovered that manually running a custom query rul
 ====
 // end::known-issue[]
 
+// tag::known-issue-53[]
+[discrete]
+.Alerts page crashes if you upgrade to 8.16 and access it in a non-default space  
+[%collapsible]
+====
+*Details* +
+On November 14, 2024, it was discovered that the Alerts page would crash if you upgraded to 8.16 and accessed the page in a non-default space.
```

Comment:
Hey @stash, I noticed, you changed `8.16.0` -> `8.16`. May i ask why? Because Error will only occur in `8.16.0` and not in `8.16.1`.

Reply:
It was just a minor style change to adhere to our writing guidelines. If we used 8.16.x, that would imply that this issue occurs across all available 8.16 releases (e.g., 8.16.1). Using 8.16 means that the issue only occurs in 8.16.0.